test/test_key_encode_decode.cpp

```
@@ -667,16 +667,20 @@ class key_factory {
   /// Used to retain arrays backing unodb::key_views.
   std::vector<std::vector<std::byte>> key_views;
 
+  UNODB_DETAIL_DISABLE_MSVC_WARNING(26481)
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If UNODB_DETAIL_DISABLE_MSVC_WARNING is a macro then please configure it.

(unknownMacro)